integration_tests/suite_test.go

```diff
@@ -165,7 +168,8 @@ var _ = BeforeSuite(func() {
 	})
 	Expect(err).NotTo(HaveOccurred())
 
-	strg, err := tokenstorage.New(ITest.Client)
+	var strg tokenstorage.TokenStorage
+	vaultTestCluster, strg = tokenstorage.CreateTestVaultTokenStorage(&testing.T{})
```

Comment:
Would `GinkgoT()` work instead of `&testing.T{}` here?

Reply:
omg, I was looking at this, thinking it is not possible because of this guy https://cs.opensource.google/go/go/+/refs/tags/go1.18:src/testing/testing.go;l=766 only now I see that vault uses it's own interface created for this exact reason https://github.com/mitchellh/go-testing-interface/blob/master/testing.go#L18.

Thanks for making me looking at it again.